File: Lecture 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_num();

int main()
{

    int n = get_positive_num();
    int space = n - 1;
    int count = 1;

    for (int i = 0; i < n; i++)
    {
        for (int k = space; k > 0; k--)
        {
            printf(" ");
        }
        for (int j = 0; j < count; j++)
        {
            printf("#");
        }

        printf("\n");
        count++;
        space--;
    }
}

int get_positive_num()
{

    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);

    return n;
}

File: Lecture_6/sentimental-mario-less/mario.py
def main():
    while True:
            try:
                n = int(input("Height: "))
                if n>0 and n<9:
                    mario(n)
                    break
                else:
                    continue
            except ValueError:
                continue


def mario(n):
    space = n-1
    count = 1

    for i in range(n):
        for j in range(space):
            print(" ", end="")
        for k in range(count):
            print("#", end="")
        print()
        space -= 1
        count += 1


main()
